config/sample.toml

```diff
@@ -93,6 +93,7 @@ format = "simple"
 # It uses the standard syscalls to perform filesystem operations
 # such as scanning the file/directory metadata.
 # This does *NOT* support per-directory quota.
+# "/vfroot" directory may not exist. Edit the directory or make it.
```

Comment:
`/vfroot` path is defined in other storage host configurations, so It might be too specific when we add this comment to local volume only. 
Why don't we move this comment to L89 instead of here?

Reply:
I didn't know that vfroot is defined like that.
I'm going to update it! thanks